common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -516,6 +523,52 @@ def verify_on_page(self, page_num, total_teams, pagination_header_text, footer_v
             msg='Expected paging footer to be ' + 'visible' if footer_visible else 'invisible'
         )
 
+    @ddt.data(
+        ('name', 'name', False),
+        ('open_slots', 'last_activity_at', True),
+        ('last_activity_at', 'open_slots', True)
+    )
+    @ddt.unpack
+    def test_sort_teams(self, sort_order, secondary_sort_order, reverse):
+        """
+        Scenario: the user should be able to sort the list of teams by name, open slots, or last activity
+        Given I am enrolled in a course with team configuration and topics
+        When I visit the Teams page
+        And I browse teams within a topic
+        Then I should see a list of teams for that topic
+        When I choose a sort order
+        Then I should see the paginated list of teams in that order
+        """
+        teams = self.create_teams(self.topic, self.TEAMS_PAGE_SIZE + 1)
+        for i, team in enumerate(random.sample(teams, len(teams))):
+            for _ in range(i):
+                user_info = AutoAuthPage(self.browser, course_id=self.course_id).visit().user_info
+                self.create_membership(user_info['username'], team['id'])
+            team['open_slots'] = self.max_team_size - i
+            # Parse last activity date, removing microseconds because
+            # the Django ORM does not support them. Will be fixed in
+            # Django 1.8.
+            team['last_activity_at'] = parse(team['last_activity_at']).replace(microsecond=0)
+        # Re-authenticate as staff after creating users
+        _ = AutoAuthPage(
```

Comment:
Nit: we generally use underscore for i18n and double underscore (``__``) for variables we don't intend to use. In this case, why do you even need to get the user info as you aren't using the variable (or am I misunderstanding how this logic works?)